src/db/spanner/models.rs

```diff
@@ -958,6 +950,8 @@ impl SpannerDb {
         }
     }
 
+    // I think we can remove this but I'm not 100% sure if the db tests use it or not.
```

Comment:
db-tests use it via two places:

- The Db trait touch_collection (which could be converted to the async version anyway)
- The test-only versions of put/post_bso which haven't been converted to async yet, so I think we still need it for now